.github/workflows/ci.yml

```
@@ -17,3 +21,23 @@ jobs:
         with:
           python-version: '3.10'
       - uses: pre-commit/action@2c7b3805fd2a0fd8c1884dcaebf91fc102a13ecd # ratchet:pre-commit/action@v3.0.1
+  test:
+    runs-on: linux-x86-g2-48-l4-4gpu
+    container:
+      # TODO: change image based on what is needed for these tests
+      image: index.docker.io/tensorflow/build@sha256:7fb38f0319bda36393cad7f40670aa22352b44421bb906f5cf34d543acd8e1d2 # ratchet:tensorflow/build:latest-python3.11
+    steps:
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # ratchet:actions/checkout@v4
+    -   name: Setup Released JAX
+        run: |
+          pip install "jax[cuda12]"
```

```suggestion
          pip install -U "jax[cuda12]"
```

This seems to be happy with the JAX v0.4.28 that is already installed, but we want the latest version. The `ml-dtypes` below installs a newer JAX but doesn't update the plugins.